.changeset/wild-seas-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ultraviolet/icons": patch
+---
+
+37 new icons in `ProductIcon`
```

Comment:
I counted 38 files, is there a duplicate or something else?

Reply:
Yes there was a duplicate (ip failover was already in product icons)

Reply:
Ok perfect then 😄